////////////////////////////////////////////////////////////////////////////////
/// @file SystemInfo.proto
///
/// @brief Device specific information
///
/// Copyright 2017 Bose Corporation
////////////////////////////////////////////////////////////////////////////////
package ProductPb;

message SystemInfo
{
    optional string name        = 1;
    optional string type        = 2;
    optional string variant     = 3;
    optional string GUID        = 4;
    optional string serialNumber= 5;
    optional string moduleType  = 6;
    optional string countryCode = 7;
    optional string regionCode  = 8;
}
